test/filename_matching.py

```diff
@@ -22,7 +22,7 @@ def check_src_files_have_test():
 def check_test_files_have_src():
     unknown_test_files = []
 
-    excluded = ['test/mitmproxy/data/', 'test/mitmproxy/net/data/', '/tservers.py']
+    excluded = ['test/mitmproxy/data/', 'test/mitmproxy/net/data/', '/tservers.py', '/conftest.py']
```

Comment:
Yes, conftest.py in the console subdirectory did. I later on removed it because it's not required anymore with the new location of the isatty check, but figured that this would be useful to leave.